### PR TITLE
feat: packageable Coral Cloud Experience Cloud site

### PR DESCRIPTION
**Pull request by pozil**
- feat: moved site to a dedicated package dir
- feat: removed no longer needed message channel
- feat: made setup idempotent
- fix: remove prettier indentation from long text content (prompt and topic fields)
- doc: formatting nit
- fix: switch Experience__c.Description__c to text to avoid formatting issue 
- feat: add member number to contact card
- feat: organize reservation insight fields
- fix: slds overflow bug

### CODE REVIEW

File: cc-base-app/main/default/lwc/experienceSchedule/experienceSchedule.js [hunk 1]
```diff
@@ -1,8 +1,6 @@
 import { LightningElement, api, wire } from 'lwc';
 import LightningAlert from 'lightning/alert';
 import { NavigationMixin } from 'lightning/navigation';
-import { subscribe, MessageContext } from 'lightning/messageService';
-import EXPERIENCE_SELECTED_MESSAGE from '@salesforce/messageChannel/ExperienceSelected__c';
```
Comment (muenzpraeger) [inline cc-base-app/main/default/lwc/experienceSchedule/experienceSchedule.js:5]: Why is this removed?

**Review by muenzpraeger — APPROVED** [review]

File: cc-base-app/main/default/lwc/experienceSchedule/experienceSchedule.js (hunk 1 above)
Comment (pozil) [reply cc-base-app/main/default/lwc/experienceSchedule/experienceSchedule.js:5]: I had to combine the experience list and the schedule components into a single component because of how Aura sites don’t deal well with split sections on their home page :grin:
Since both component share ancestry, I’ve used the recordId API attribute to pass the selection. This way it’s consistent with what we have in the Org.
